src/components/ManaCalculator/components/OutputForm.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { useManaState, useResults } from '../hooks';
 import { fromMicro, roundMax } from '../utils';
+import { UserType } from '../enums';
 import humanizeDuration from 'humanize-duration';
 
 export function OutputForm() {
   const { state } = useManaState();
   const results = useResults(state);
   const passiveRewards = roundMax(fromMicro(results.passiveRewards), 6);
   const manaGenerated = roundMax(fromMicro(results.generatedRewards), 6);
-  const totalBPS = roundMax(results.totalBPS, 2);
+  const blockAllowance = roundMax(results.blockAllowance, 2);
+  const totalBPS = roundMax(results.totalBPS, 20);
```

Comment:
Do we really want to have up to 20 decimal numbers? Perhaps you wanted to use `2` ? @oliviasaa